client/commands/v2/pysa_server.py

```diff
@@ -60,6 +65,56 @@ def __init__(
         self.binary_location = binary_location
         self.server_identifier = server_identifier
 
+    def invalid_model_to_diagnostic(
+        self,
+        invalid_model: query.InvalidModel,
```

Comment:
pyre complains that `query.InvalidModel` does not exist. Any idea why?

Reply:
That seems strange. I'm not sure why it says that.